Shared/Samples/Show utility associations/ShowUtilityAssociationsView.swift

```diff
@@ -0,0 +1,263 @@
+        }
+        .task {
+            await model.setup()
+            try? await model.addAssociationGraphics(viewpoint: viewpoint, scale: scale)
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                legend
+            }
+        }
+    }
+}
+
+private extension ShowUtilityAssociationsView {
+    /// The legend at the bottom of the screen.
+    var legend: some View {
+        HStack {
+            Label {
+                Text("Attachment")
+            } icon: {
+                if let attachmentImage {
+                    Image(uiImage: attachmentImage)
+                } else {
+                    Color.clear
+                }
+            }
+            .task(id: displayScale) {
+                attachmentImage = try? await Symbol.attachment
+                    .makeSwatch(scale: displayScale)
+            }
+            Spacer()
+            Label {
+                Text("Connectivity")
+            } icon: {
+                if let connectivityImage {
+                    Image(uiImage: connectivityImage)
+                } else {
+                    Color.clear
+                }
+            }
+            .task(id: displayScale) {
+                connectivityImage = try? await Symbol.connectivity
+                    .makeSwatch(scale: displayScale)
+            }
+        }
+        .labelStyle(.titleAndIcon)
+    }
+}
+
+private extension ShowUtilityAssociationsView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// The map with the utility network.
+        let map = Map(basemapStyle: .arcGISTopographic)
+        
+        /// The utility network for this sample.
+        private let network = UtilityNetwork(url: .utilityNetwork)
+        
+        /// A container for associations results.
+        let associationsOverlay = makeAssociationsOverlay()
+        
+        /// A Boolean value indicating if the sample is authenticated.
+        private var isAuthenticated: Bool {
+            !ArcGISEnvironment.authenticationManager.arcGISCredentialStore.credentials.isEmpty
+        }
+        
+        /// The max scale for the viewpoint.
+        private var maxScale: Double { 2_000 }
+        
+        init() {
+            map.initialViewpoint = .initialViewpoint
+            map.addUtilityNetwork(network)
+        }
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        // MARK: Methods
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        func setup() async {
+            try? await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+            try? await network.load()
```

Comment:
This isn't quite what I meant. I meant this:

```suggestion
        func setup() async throws {
            try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
            try await network.load()
```

And then the caller should either handle the thrown error or discard it:

```swift
try? model.setup
```